src/api/app/controllers/tag_controller.rb

```diff
@@ -146,11 +144,9 @@ def get_tags_by_user_and_package( do_render = true )
     @project = @package.project
 
     @tags = @package.tags.where("taggings.user_id = ?", user.id).order(:name)
-    if do_render
-      render partial: "tags"
-    else
-      return @tags
-    end
+    return @tags unless do_render
+
```

Comment:
because i felt like it? lol, i think it looks nicer but its not a big deal so i will get rid of them..